src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+
+public class Ui {
+
+    /***
+     * Print the opening menu
```

Comment:
Nitpicking but consider using prints instead of print. Same for other javadoc comments :D